mirgecom/filter.py

```diff
@@ -0,0 +1,200 @@
+""":mod:`mirgecom.filter` is for filters and filter-related constructs.
```

Comment:
This should have an `.. automodule::` somewhere so that it's included in the docs.

Reply:
Done: https://github.com/illinois-ceesd/mirgecom/pull/324/commits/b6002f806d01c7a0df9b2b1775c2d81044005301